src/controllers/wallet/wallet.controller.js

```diff
@@ -457,11 +468,19 @@ async function meltTokens(req, res) {
   const { token, amount } = req.body;
   const changeAddress = req.body.change_address || null;
   const depositAddress = req.body.deposit_address || null;
+  const meltAuthorityAddress = req.body.melt_authority_address || null;
+  const allowExternalMeltAuthorityAddress = req.body.allow_external_melt_authority_address || false;
+
   try {
     const response = await wallet.meltTokens(
       token,
       amount,
-      { address: depositAddress, changeAddress }
+      {
+        address: depositAddress,
+        changeAddress,
+        meltAuthorityAddress,
+        allowExternalMeltAuthorityAddress
```

Comment:
This parameter might be renamed. I'm creating the thread so you check it before merging.

Reply:
Nothing to do here. Just closing this conversation.